sharepoint/sharepoint-ps/sharepoint-online/Start-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-sporestrictedaccessforsitesinsights
+applicable: SharePoint Online
+title: Start-SPORestrictedAccessForSitesInsights
+schema: 2.0.0
+author: itambakhe
+ms.author: itambakhe
+ms.reviewer:
+manager:
+---
+ 
+# Start-SPORestrictedAccessForSitesInsights
+
+## SYNOPSIS
+
+This cmdlet enables administrator to trigger the build of a new Restricted Access Control insights report for the data from last 28 days.
```

Comment:
>Restricted Access Control [](http://example.com/codeflow?start=64&length=25)

is this the right casing? the other file does it differently

Reply:
Made changes to follow the "restricted access control" casing in both the files.